Latin Culture Group

Charanraja123 opened the pull request:
Added all basic ruler titles with english localisations

Charanraja123 commented:
Do not mind Italian culture and its clashes with my edits, Since its a modification I'll clean it up in Fixes later.
Still yet to add titles queen mother type and consorts. Will be added by tomorrow at latest.
Only did English localization, when approved will add other localization too
I have kept the cultures separate, as there are very titles that are same. Even those which are same are not within their tier. For example, Baron might be same but Baroness might differ kind of thing. Therefore it was easier for me to just split them up and mod them in instead of trying to organise them into groups.
I've tried to avoid mistakes as much as possible, but please check and let me know.

Gr770 commented:
Does tier = { kingdom empire } work?

Also make sure to copy the localization from english to the other latin localizations.

Charanraja123 commented:
I think tier = {kingdom empire} was used before and generated no errors (I may be wrong). In any case, I'll test it later again today.

Regarding other localisations, I'm waiting for everyone to be okay with modded files, then I can copy it to all languages. I do not want to be making changes across four language localisations.

Viridianus commented on common/flavorization/00_CTP.txt:
```diff
@@ -6868,6 +6868,1335 @@ duke_tribal_female_italian = {
+	type = character
+	gender = female
+	special = ruler_child
+	tier = { kingdom empire }
```
Does this really work? I'd expect that since tier = kingdom does not have curly brackets, you can't set a set here.

Viridianus replied:
Oh, @Gr770 asked the same thing, funny)

Charanraja123 replied:
Doesn't work. Changed it.

Viridianus commented on localization/english/culture/ctp_titles_l_english.yml:
```diff
@@ -414,6 +414,144 @@
+ count_feudalclan_male_lombard:0 "Gastaldus"
+ count_feudalclan_female_lombard:0 "Gastaldissa"
+ county_feudalclan_lombard:0 "Gastaldia"
+ count_feudalclan_male_lombard_independent:0 "Princeps"
```
Am I the only one who's weirded out by Latin titles for Lombard?

Gr770 replied:
I believe lombard is dead and not used save for historic rulers.

Charanraja123 replied:
Nope. I just went through some of province files. Everything in the "d_spoleto" duchy has the lombard culture for 867 start.

Viridianus replied:
Then we should double-think it probably - I can't believe they retained proper declensions of Latin.

Charanraja123 replied:
Lombard is as good as non-existent now

Viridianus commented on localization/english/culture/ctp_titles_l_english.yml:
```diff
@@ -414,6 +414,144 @@
  duke_tribal_female_italian:0 "Gran Capo"
 
+ ##Roman##
+ baron_male_roman:0 "Baron"
```
Almost certainly a mistake in Master Sheet.

Charanraja123 replied:
You can edit any localisations directly by yourself which are wrong. 
I Just added in what was already present in the sheets and have absolutely no knowledge on my own regarding this, so I give you full clearance to go bat shit crazy on my localisations file

Viridianus commented on localization/english/culture/ctp_titles_l_english.yml:
```diff
@@ -414,6 +414,144 @@
 
+ ##Roman##
+ baron_male_roman:0 "Baron"
+ baron_female_roman:0 "Barona"
```
A mistake in Master Sheet, should be Baronissa

Viridianus reviewed (changes requested):
I'm not at all sure about Lombard; please limit it to the others for now and try to incorporate my other notes.

Gr770 commented:
@Charanraja123 Works for me

Charanraja123 commented:
@Viridianus I'm going to change the priority numbers for all of Lombard culture to 1, so that they never get called. In case we get the write title, we could just change the numbers and use them again. I'll check the tier stuff now,, Sorry this took so long. I got preoccupied with the custom localisation thing

Charanraja123 commented:
I Removed that tier stuff. Doesnt work. Split them into empire and kingdom. Retained the same priority as title prince is still considered a lesser title king. 
Also set alll lombard definitions priority to one, so they have no chance of getting called.
I hope everything is okay now
